### ISSUE TITLE
swc_ecma_visit skips class body

### ISSUE THREAD

**Issue by bartlomieju** [swc-project/swc#808]
**Describe the bug**
`swc_ecma_visit` doesn't visit class body

**Input code**
```rust
impl Visit for RequireYieldVisitor {
  ... 
  fn visit_class(&mut self, class: &Class, parent: &dyn Node) {
    eprintln!("visit class");
    swc_ecma_visit::visit_class(self, class, parent);
  }

  fn visit_class_method(&mut self, class_method: &ClassMethod, parent: &dyn Node) {
    eprintln!("visit class method");
    swc_ecma_visit::visit_class_method(self, class_method, parent);
  }
}
```

```shell
$ cargo test require_yield
visit class
```

**Expected behavior**
```shell
$ cargo test require_yield
visit class
visit class method
```

**Version**
`swc_ecma_visit` 0.5.0

**Comment by kdy1** [comment]
Hmm... it's quite interesting. I'll try to fix it asap.

**Comment by bartlomieju** [comment]
@kdy1 thanks, I was surprised as well - when I print out AST node of class I can see body containing method, but `visit_class_method()` is never called.

**Comment by kdy1** [comment]
@bartlomieju Can you share your input code?

**Comment by bartlomieju** [comment]
@kdy1 full repro in this PR: https://github.com/denoland/deno_lint/pull/56

You can run problematic test by: `cargo test require_yield`

**Comment by kdy1** [comment]
Published `swc_ecma_visit@0.5.1`

**Comment by bartlomieju** [comment]
@kdy1 thank you!
